File: Java/Ram.java
public class Ram {
    private int kapasitasGB;
    private String ddr;
    private String merk;
    private String nama;

    public Ram(int kapasitasGB, String ddr, String merk, String nama) {
        this.kapasitasGB = kapasitasGB;
        this.ddr = ddr;
        this.merk = merk;
        this.nama = nama;
    }

    public int getKapasitasGB() { return kapasitasGB; }
    public String getDdr() { return ddr; }
    public String getMerk() { return merk; }
    public String getNama() { return nama; }
}

File: Python/Ram.py
from Komponen import Komponen

class Ram(Komponen):
    def __init__(self, kapasitas_gb, ddr, merk, nama):
        super().__init__(merk, nama)
        self._kapasitas_gb = kapasitas_gb
        self._ddr = ddr

    def get_kapasitas_gb(self):
        return self._kapasitas_gb

    def set_kapasitas_gb(self, kapasitas_gb):
        self._kapasitas_gb = kapasitas_gb

    def get_ddr(self):
        return self._ddr

    def set_ddr(self, ddr):
        self._ddr = ddr
